docs/usage/shoot_supported_architectures.md

```diff
@@ -0,0 +1,21 @@
+# Supported CPU Architectures for Shoot Worker Nodes
+
+Users can create shoot clusters with worker groups having virtual machines of different architectures. CPU architecture of each worker pool can be specified in the `Shoot` specification as follows:-
+
+## Example Usage in a `Shoot`
+
+```yaml
+spec:
+  provider:
+    workers:
+    - name: cpu-worker
+      machine:
+        architecture: <some-cpu-architecture>
+```
+
+If no value is specified for the architecture field, it defaults to `amd64`. For a valid shoot object, a machine should be present in the respective CloudProfile with the same CPU architecture as specified in the `Shoot` yaml.
```

Comment:
```suggestion
If no value is specified for the architecture field, it defaults to `amd64`. For a valid shoot object, a machine type should be present in the respective `CloudProfile` with the same CPU architecture as specified in the `Shoot` yaml.
```

Don't we need a machine image with the right architecture as well?
Can you maybe also show some short YAML example of such `CloudProfile`?